project/plugins.sbt

```diff
@@ -1,27 +1,13 @@
+val sbtTypelevelVersion = "0.4.7"
+addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % sbtTypelevelVersion)
+addSbtPlugin("org.typelevel" % "sbt-typelevel-settings" % sbtTypelevelVersion)
+addSbtPlugin("org.typelevel" % "sbt-typelevel-site" % sbtTypelevelVersion)
```

Comment:
I forgot to mention: in the end, I didn't actually migrate to sbt-typelevel since that would require adding copyright headers to all the files. Instead we use a combination of sbt-typelevel-ci-release and sbt-typelevel-settings. So the build still has a little more boilerplate than it could, but it's okay.

Reply:
I don't see a particular reason not to bite the bullet and add the headers, though it would be nice to get this substantial work to a landing.

Reply:
Yep, super easy to make that change. Mostly it just creates a massive diff :)

Also, I wasn't completely certain what to make of Cats' [COPYING](https://github.com/typelevel/cats/blob/main/COPYING) which includes a Scalaz notice as well. E.g. whether it's appropriate to put a blanket license over every file.

Reply:
In http4s, we generate our copyright and also preserve the original copyright on derived files.  This is a fairly common practice, and one that doesn't irritate the tooling.

Reply:
Right, http4s has been very diligent about that which helped. But AFAICT not a single file in Cats has a copyright header, whether its derived or not.

Reply:
Oh, I see.  I don't know how we'd go back and add that now.  I say we made a good faith effort with the blanket COPYING file, and I'd be happy to add a second header if anyone cares enough to point out the derived works.